README.md

```diff
@@ -50,23 +45,7 @@
 
 ### Key Features <a name="key-features"></a>
 
-- **Add new categories**
-- **Add new transactions**
-- **See amount of all transactions**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
-### 🚀 Live Demo <a name="live-demo"></a>
-
-- [Budget Tracker](https://budget-tracker-azmz.onrender.com)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-
-### 📽️ Video Presentation <a name="video"></a>
-
-- [Video link](https://www.loom.com/share/1213a4eea2784cfe924a89b3ab8dd3ba)
+- **Add new greetings**
 
```

Comment:
- [ ] [OPTIONAL]
Kindly ensure that the _Getting Started_ section_ is filled with the appropriate instructions on how to _run_ your app, on _line 54_.